src/typeGuards.ts

```diff
@@ -0,0 +1,5 @@
+// (C) 2007-2019 GoodData Corporation
+import { IGuidColorItem, IRGBColorItem, IColorItem } from './interfaces';
+
+export const isGuidColorItem = (color: IColorItem): color is IGuidColorItem => color.type === 'guid';
```

Comment:
I think also typeguard should have tests...what happens when `color` will be undefined? Should it throw error or should it return false?

Reply:
The project is compiled with `"strictNullChecks": true` so calling `isGuidColorItem(undefined)` results in a compilation error. However, I made it so that they are more similar to the type guards in `@gooddata/typings` and added tests for them. Thanks for pointing this out.